config.json

```diff
@@ -19,6 +19,7 @@
       {
         "slug": "entry-api",
         "uuid": "e7d8603e-97a9-11ea-9753-0242ac110002",
+				"name": "Entry API",
```

Comment:
These render weirdly because it looks like you used tabs instead of spaces, or something. Please update to an appropriate number of spaces.